src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/bail/adminofficer/email/AdminOfficerBailSummaryUploadedPersonalisation.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.bail.adminofficer.email;
+
+import static java.util.Objects.requireNonNull;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+import javax.validation.constraints.NotNull;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.BailCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.BailCaseFieldDefinition;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.BailEmailNotificationPersonalisation;
+import uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.EmailAddressFinder;
+
+@Service
+public class AdminOfficerBailSummaryUploadedPersonalisation implements BailEmailNotificationPersonalisation {
+
+    private final String bailSummaryUploadedAdminOfficerTemplateId;
+    private final EmailAddressFinder emailAddressFinder;
+
+    public AdminOfficerBailSummaryUploadedPersonalisation(
+            @NotNull(message = "bailSummaryUploadedAdminOfficerTemplateId cannot be null")
+            @Value("${govnotify.template.bail.uploadBailSummary.email}") String bailSummaryUploadedAdminOfficerTemplateId,
+            EmailAddressFinder emailAddressFinder
+    ) {
+        this.bailSummaryUploadedAdminOfficerTemplateId = bailSummaryUploadedAdminOfficerTemplateId;
+        this.emailAddressFinder = emailAddressFinder;
+    }
+
+    @Override
+    public String getTemplateId() {
+        return bailSummaryUploadedAdminOfficerTemplateId;
+    }
+
+    @Override
+    public Set<String> getRecipientsList(BailCase bailCase) {
+        return Collections.singleton(emailAddressFinder.getBailHearingCentreEmailAddress(bailCase));
+    }
+
+    @Override
+    public String getReferenceId(Long caseId) {
+        return caseId + "_BAIL_SUMMARY_UPLOADED_ADMIN_OFFICER";
+    }
+
+    @Override
+    public Map<String, String> getPersonalisation(BailCase bailCase) {
+        requireNonNull(bailCase, "bailCase must not be null");
+
+        return ImmutableMap
+                .<String, String>builder()
+                .put("bailReferenceNumber", bailCase.read(BailCaseFieldDefinition.BAIL_REFERENCE_NUMBER, String.class).orElse(""))
+                .put("legalRepReference", bailCase.read(BailCaseFieldDefinition.LEGAL_REP_REFERENCE, String.class).orElse(""))
```

Comment:
Check with Elli and Ryan, for scenario when HO/Admin submits Application with no LR, this field will appear blank. I think we need to create this as Optional Content. Ryan is working on something similar on Submit Application event.